sass/directives/02_base_components/button/_button.scss

```diff
@@ -222,6 +222,101 @@ $massive-button-horizontal-padding: $base-input-font-size * 2;
   }
 }
 
+// Fancy Play Button
+// Initial state
+$ripple-diameter: 24em;
```

Comment:
no, going forward this is something I'd prefix with an `_` and leave it in the file where it's used